RedHat-UX/red-hat-design-system
# docs: sort elementDocs alphabetically by slug

## What I did

1. We're using the `elementDocs` collection when generating the side nav, but it's not created/sorted with the other collections used in the `RHDSAlphabetizeTagsPlugin`. Before returning, I'm sorting `elementDocs` by the slug since it takes a component's alias into consideration. 

## Testing Instructions

1. Run the docs with `npm run start` locally and verify that the elements are listing in alphabetical order in the side navigation under `Elements`.
